Data/ByteString/Builder/Internal.hs

```diff
@@ -414,6 +417,12 @@ instance Monoid Builder where
   {-# INLINE mconcat #-}
   mconcat = foldr mappend mempty
 
+instance IsList Builder where
+  type Item Builder = Word8
+  fromList = byteStringInsert . fromList
+  fromListN n = byteStringThreshold n . fromList
+  toList = toList . toLazyByteStringWith (safeStrategy L.smallChunkSize L.smallChunkSize) mempty
```

Comment:
Is there any particular reason to prefer this over just `toList . toLazyByteString`? Ah, for whatever reason the definition of `toLazyByteString` is not in this module. Let's just move the latter here so we can use it directly.